Assets/MRTK/SDK/Features/UX/Scripts/PressableButtons/PressableButton.cs

```diff
@@ -425,8 +425,10 @@ void IMixedRealityTouchHandler.OnTouchCompleted(HandTrackingInputEventData event
         /// </summary>
         /// 
         public Vector3 GetWorldPositionAlongPushDirection(float localDistance)
-        {
-            float distance = (distanceSpaceMode == SpaceMode.Local) ? localDistance * LocalToWorldScale : localDistance;
+        {   
+            // We clamp the calculated local distance to the maxPushDistance,
+            // because LocalToWorldScale may return NaNs when object scale is zero.
+            float distance = Mathf.Clamp(0f, maxPushDistance, (distanceSpaceMode == SpaceMode.Local) ? localDistance * LocalToWorldScale : localDistance);
```

Comment:
I think the parameters for `Clamp` are slightly switched around here. It looks like it's [`Clamp(value, min, max)`](https://docs.unity3d.com/ScriptReference/Mathf.Clamp.html), but this looks like it's using it as `Clamp(min, max, value)`.

Reply:
Yeah the distance remains constant because the values are a little mixed up

![pressDistanceGif](https://user-images.githubusercontent.com/53493796/82476484-01cfa480-9a83-11ea-9065-d8af85757848.gif)